tasks/compilers.yml

```diff
@@ -20,7 +20,23 @@
     mode: "0750"
     state: file
     follow: true
-  with_items:
-    - "{{ compiler.files }}"
+  loop: "{{ compiler.files }}"
+  tags:
+    - compilers
+
+- name: Ensure restrict compilers access via dpkg-statoverride
+  become: true
+  ansible.builtin.command: dpkg-statoverride --update --force-all --add root root 0750 "{{ item.path }}"
+  register: dpkg_statoverride
+  changed_when: dpkg_statoverride.rc != 0
+  loop: "{{ compiler.files }}"
+  when: ansible_os_family == 'Debian'
+  tags:
+    - compilers
+
+- name: Ensure restrict compilers access via DNF post-transaction-actions Plugin
+  ansible.builtin.include_tasks: compilers_dnf_post_transaction_actions_plugin.yml
+  loop: "{{ compiler.files | map(attribute='path') | unique }}"
+  when: ansible_os_family == 'RedHat'
```

Comment:
Why was this added?

Reply:
I have continued to backport commits from upstream. See https://github.com/cisagov/ansible-role-hardening-2/commit/0ccbf12f1d214c61abd77db5baf5ede724c3476e

Reply:
The way to backport commits from upstream is to either:
- `git rebase` on top of the upstream
- `git cherry-pick` from upstream

You shouldn't recreate the commits yourself.

Reply:
I'm always learning